src/launcher/standalone_launcher/install/README.md

```diff
@@ -0,0 +1,60 @@
+# Installers
```

Comment:
Delete the `install` directory for now. We have all the installers in src/client at the moment. We will revisit this later.